tests/unit/api/auth.py

```diff
@@ -95,68 +95,68 @@ def server_init():
 auth_call_params = [
     # network_create
 
-    ### Legal cases:
```

Comment:
It would accept ``# ##``, but it would be pretty ugly that way. What about 
```
#
# network_create
#

# Legal cases:

# Admin creates a public network internal to HaaS:
dict(fn=api.network_create,
     error=None,
     admin=True,
     project=None,
     args=['pub', 'admin', '', '']),
```
// @zenhack